extensions/wikia/TrackingOptIn/js/trackingOptIn.js

```diff
@@ -44,9 +60,54 @@ define('wikia.trackingOptIn', [
 		window.Wikia.consentQueue.push(callback);
 	}
 
+	function geoRequiresTrackingConsent() {
+		return geoRequiresConsent;
+	}
+
+	function getPrebidGlobalVendorList() {
+		var request = new XMLHttpRequest();
+
+		log('Requesting IAB Global Vendor List', log.levels.debug, logGroup);
+
+		request.open('GET', prebidVendorListUrl, false);
```

Comment:
I have doubts about requesting this resource directly from `consensu.org`. Also - what happens in case of failure/adblock?

Reply:
Turns out requesting directly is not a good idea - this server is often unstable. List trimmed and copied into our code.